Hello Conversion Layer Adapter (#1976)
cfug/dio: plugins/compatibility_layer/README.md

# dio_compatibility_layer

[![pub package](https://img.shields.io/pub/v/dio_compatibility_layer.svg)](https://pub.dev/packages/dio_compatibility_layer)
[![likes](https://img.shields.io/pub/likes/dio_compatibility_layer)](https://pub.dev/packages/dio_compatibility_layer/score)
[![popularity](https://img.shields.io/pub/popularity/dio_compatibility_layer)](https://pub.dev/packages/dio_compatibility_layer/score)
[![pub points](https://img.shields.io/pub/points/dio_compatibility_layer)](https://pub.dev/packages/dio_compatibility_layer/score)

If you encounter bugs, consider fixing it by opening a PR or at least contribute a failing test case.

> This is adapters for [Dio](https://pub.dev/packages/dio)
> which enables you to make use of other HTTP clients as underlying implementation.
> 
> Supported client base currently:
> - `http`

Author: ```suggestion
This package contains adapters for [Dio](https://pub.dev/packages/dio)
which enables you to make use of other HTTP clients as the underlying implementation.

Currently, it supports compatibility with
- [`http`](https://pub.dev/packages/http)
```
